### ISSUE TITLE
ENS as TLD?

### ISSUE THREAD

**Issue by smutt** [nweaver-blockchain/blockchain-names#6]
"Overall Suitability: ENS has several limitations that may prove problematic if it seeks to act as a significant TLD."

I don't understand what the word TLD means here. Is this a typo? Do you mean it will be hard to have ENS replace the DNS?

**Comment by nweaver-blockchain** [comment]
No, ENS proposes to be a top level domain (like .com) rather than replacing the root.

**Comment by smutt** [comment]
But ENS is still not a TLD. It's not even a domain name. It's a protocol, or a namespace, or maybe an implementation.

Do you mean to say ENS wishes to be a registry of names similar to what we have in the DNS at the TLD level?
